package.json

```diff
@@ -14,95 +14,95 @@
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-react-hmre": "^1.1.1",
-    "babel-preset-stage-0": "^6.24.1",
-    "babel-runtime": "^6.23.0",
-    "chai": "^4.0.2",
-    "chai-enzyme": "^0.7.1",
-    "cheerio": "^1.0.0-rc.1",
+    "@babel/cli": "^7.0.0-beta.32",
+    "@babel/core": "^7.0.0-beta.32",
+    "@babel/plugin-transform-react-display-name": "^7.0.0-beta.32",
+    "@babel/plugin-transform-runtime": "^7.0.0-beta.32",
+    "@babel/polyfill": "^7.0.0-beta.32",
+    "@babel/preset-env": "^7.0.0-beta.32",
+    "@babel/preset-react": "^7.0.0-beta.32",
+    "@babel/preset-stage-0": "^7.0.0-beta.32",
+    "@babel/runtime": "^7.0.0-beta.32",
+    "autoprefixer": "^7.1.6",
+    "babel-eslint": "^8.0.2",
+    "babel-loader": "^8.0.0-beta.0",
+    "chai": "^4.1.2",
+    "chai-enzyme": "^1.0.0-beta.0",
+    "cheerio": "^1.0.0-rc.2",
     "classnames": "^2.2.5",
-    "css-loader": "^0.28.4",
-    "debug": "^2.6.8",
-    "enzyme": "^2.9.1",
-    "eslint": "^4.1.0",
+    "css-loader": "^0.28.7",
+    "debug": "^3.1.0",
+    "enzyme": "^3.2.0",
+    "enzyme-adapter-react-16": "^1.1.0",
+    "eslint": "^4.11.0",
     "eslint-config-standard": "^10.2.1",
     "eslint-config-standard-react": "^5.0.0",
-    "eslint-plugin-import": "^2.6.0",
-    "eslint-plugin-node": "^5.1.0",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-react": "^7.1.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-react": "^7.5.1",
     "eslint-plugin-standard": "^3.0.1",
     "exports-loader": "^0.6.4",
-    "file-loader": "^0.11.2",
-    "hjs-webpack": "^9.1.0",
+    "file-loader": "^1.1.5",
+    "hjs-webpack": "https://github.com/hacdias/hjs-webpack/tarball/master",
     "https-browserify": "^1.0.0",
     "imports-loader": "^0.7.1",
-    "json-loader": "^0.5.4",
-    "karma": "^1.7.0",
+    "json-loader": "^0.5.7",
+    "karma": "^1.7.1",
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "^1.0.1",
     "karma-mocha": "^1.3.0",
     "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^2.0.3",
-    "less": "^2.7.2",
-    "less-loader": "^4.0.4",
-    "mocha": "^3.4.2",
-    "postcss-loader": "^2.0.6",
+    "karma-webpack": "^2.0.6",
+    "less": "^2.7.3",
+    "less-loader": "^4.0.5",
+    "mocha": "^4.0.1",
+    "postcss-loader": "^2.0.8",
     "pre-commit": "^1.2.2",
-    "react-addons-test-utils": "^15.6.0",
-    "react-test-renderer": "^15.6.1",
+    "react-addons-test-utils": "^15.6.2",
+    "react-test-renderer": "^16.1.1",
     "redux-logger": "^3.0.6",
-    "standard": "^10.0.2",
+    "standard": "^10.0.3",
     "stream-http": "^2.7.2",
-    "style-loader": "^0.18.2",
-    "url-loader": "^0.5.9",
-    "webpack": "^3.0.0"
+    "style-loader": "^0.19.0",
+    "url-loader": "^0.6.2",
+    "webpack": "^3.8.1"
   },
   "pre-commit": [
     "lint"
```

Comment:
Sooo greeeen! :D

Reply:
Am I supposed to change anything in here? 😮

Reply:
just happy :)